Marlin/src/inc/Conditionals_post.h

```diff
@@ -2419,6 +2419,11 @@
  * Heater signal inversion defaults
  */
 
+struct HeaterPin {
+  pin_t pin;
+  bool  inverting;
+};
```

Comment:
This should be defined elsewhere, and `typedef struct { ... } heater_pin_t;` would be more like the common idiom.

Reply:
Ok. I'm seen both usage in the code (struct definition with and without typedef), and the typedef struct looks like C code to me. It has numerous drawback, like you can't have a constructor in a typedef struct since it's unamed. I'll move it back to temperature.h.